mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -900,6 +944,78 @@ class MqttSessionSpec
       }
     }
 
+    "receive two subscriptions for the same topic" in {
```

Comment:
Additional test - nothing directly to do with the problem being solved by the PR